data/shaders/opengl/sgaussian.glsl

```diff
@@ -0,0 +1,35 @@
+// Copyright © 2008-2021 Pioneer Developers. See AUTHORS.txt for details
+// Licensed under the terms of the GPL v3. See licenses/GPL-3.txt
+
+// Code adapted from https://therealmjp.github.io/posts/sg-series-part-2-spherical-gaussians-101/
+
+struct SGaussian {
```

Comment:
I am a little unsure as to which license to use for this code; I could not find a license statement on the blog so I took the examples to be public domain / uncopyrighted and considered the GLSL version to be enough of a derivative work to qualify for the Pioneer copyright.